src/components/eholdings/eholdings.js

```diff
@@ -98,6 +98,7 @@ class EHoldings extends Component {
       match: { path: rootPath },
     } = this.props;
 
+
```

Comment:
Why do we need two empty lines here?

Reply:
It' need just to see on sonar coverage percentage. I'll remove it